src/api/app/views/webui/users/edit.html.haml

```diff
@@ -8,11 +8,11 @@
         = form.hidden_field(:login)
         .mb-3
           = form.label(:realname, 'Name:')
-          = form.text_field(:realname, disabled: user_is_configurable(@displayed_user), class: 'form-control')
+          = form.text_field(:realname, disabled: !user_is_configurable(@displayed_user), class: 'form-control')
```

Comment:
You are changing the logic here, why? I don't understand...

Reply:
Yeah... I'll revert those changes.

Reply:
I removed the `user_is_configurable` method. Now that we no longer use LDAP for authentication, it doesn't make much sense to keep it.